### PR TITLE
release: Bump version to v0.16.3-0.5.0

### PR DESCRIPTION
**Pull request by loloicci**
# Description

fix devtools/set_version.sh and bump version.

## Types of changes

- [ ] Bug fix (changes which fixes an issue)
- [ ] New feature (changes which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] ETC (build, ci, docs, perf, refactor, style, test)

## Checklist

- [ ] I followed the [contributing guidelines](https://github.com/line/cosmwasm/blob/main/CONTRIBUTING.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] The PR title and commits are followed [conventional commit form](https://www.conventionalcommits.org/en/v1.0.0).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+### Features
+
+* Add derive macro "IntoEvent" ([#161](https://github.com/line/cosmwasm/issues/161))
+* merge original version 0.16.3 ([#148](https://github.com/line/cosmwasm/issues/148))
+* pin dependent uint version 0.9.1
+* Merge branch 'cosmwasm-0-16-2' into versionup-work
+* support Uuid type and sha1_calculate API ([#145](https://github.com/line/cosmwasm/issues/145))
+* Add release automation config ([#108](https://github.com/line/cosmwasm/issues/108))
+
+### Fixes
```
Comment (brew0722) [inline CHANGELOG.md:15]: How about writing it based on PRs? As in the other versions below.

Comment (loloicci) [reply CHANGELOG.md:15]: This is generated by "devtools/update_changelog.sh"

Comment (brew0722) [reply CHANGELOG.md:15]: Oh... Well, then, please manage once manually.
we need formatting, but we haven't time..

Comment (loloicci) [reply CHANGELOG.md:13]: OK, updated.

**Review by zemyblue — APPROVED** [review]

**Review by shiki-tak — APPROVED** [review]

**Review by brew0722 — APPROVED** [review]

**Pull request merged**